docs/src/pages/customization/how-to-customize/DynamicClassName.tsx

```diff
@@ -53,3 +56,11 @@ export default function DynamicClassName() {
     </React.Fragment>
   );
 }
+
+export default function DynamicClassName() {
```

Comment:
Do we even want to keep this demo? Looking at https://next--material-ui.netlify.app/customization/how-to-customize/#class-name-branch, I'm not sure if it's relevant.

Reply:
Good point, I think this was mostly leftover from the v4 docs page. I'll remove it.